### PR TITLE
Update to containers/image/v5

### PR DESCRIPTION
**Pull request by rhatdan**
> gomove github.com/containers/image/v4 github.com/containers/image/v5
> GO111MODULE=on go mod edit -replace github.com/containers/image/v5=github.com/mtrmac/image/v5@release-5
> GO111MODULE=on go get github.com/containers/image/v5
> make vendor

Replaces: https://github.com/containers/skopeo/pull/747

Signed-off-by: Miloslav Trmač <mitr@redhat.com>
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ matrix:
      sudo: required
      services:
        - docker
+     before_install:
+         - sudo add-apt-repository -y ppa:projectatomic/ppa
+	 - sudo apt-get update
+	 - sudo podman
```
Comment (TomSweeneyRedHat) [inline .travis.yml]: space/tab issues?

**Comment by TomSweeneyRedHat** [comment]
5 hour test cycle is a little odd, it's still cranking.
Do we need to bump golang to 1.13?

**Comment by vrothberg** [comment]
Travis seems to have an issue reporting. I restarted the tests to look into it.

Regarding the PR. I would have preferred to split the `make vendor-in-container` from the v5 vendoring. They're not strictly related and one could block the other.

**Comment by vrothberg** [comment]
The CI is still calling `make vendor` it seems. I'll open a PR for vendoring image v5. We can use this one here to the `vendor-in-container` work.

**Comment by vrothberg** [comment]
v5 vendoring https://github.com/containers/skopeo/pull/749

**Pull request closed without merging**